knowledge-base/dropdownlist-custom-sorting-order-groups.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Custom Sorting Order for Groups in DropDownList
+description: Learn how to apply a custom order to groups in the DropDownList for Blazor.
+type: how-to
+page_title: How to Set a Custom Grouping Order in Blazor DropDownList
+slug: dropdownlist-custom-sorting-order-groups
+tags: dropdownlist, blazor, telerik, grouping, custom order
+res_type: kb
+ticketid: 1666981, 1579415
+---
+
+## Environment
+
+<table>
+<tbody>
+<tr>
+<td>Product</td>
+<td>DropDownList for Blazor</td>
+</tr>
+<tr>
+<td>Version</td>
+<td>6.2.0</td>
```

Comment:
Do we need to explicitly list the version here as we do not do that in the general case? This may give the impression that the article is only applicable to v 6.2.0 which I believe is not the case.
If we want to mark the version as of which the grouping is available, we can set [2.25](https://www.telerik.com/support/whats-new/blazor-ui/release-history/ui-for-blazor-2-25-0) and up.